aries_vcx_core/src/ledger/indy_vdr_ledger.rs

```diff
@@ -371,26 +376,23 @@ where
         &self,
         schema_json: &str,
         submitter_did: &str,
-        endorser_did: Option<String>,
+        _endorser_did: Option<String>,
```

Comment:
well it is improvement, but since the PR state name is "implement tx endoring ..." I think actually doing the endorsement would be within the scope of PR

Reply:
Added.